Templates/templates/CSharp/Requests/EntityCollectionRequest.cs.tt

```diff
@@ -46,26 +46,10 @@ namespace <#=@namespace#>
         {
             this.Method = <#=templateWriter.GetMethod#>;
             var response = await this.SendAsync<<#=collectionResponse#>>(null, cancellationToken).ConfigureAwait(false);
-            if (response != null && response.Value != null && response.Value.CurrentPage != null)
+            if (response?.Value?.CurrentPage != null)
             {
-                if (response.AdditionalData != null)
-                {
-                    object nextPageLink;
-                    response.AdditionalData.TryGetValue("@odata.nextLink", out nextPageLink);
-
-                    var nextPageLinkString = nextPageLink as string;
-
-                    if (!string.IsNullOrEmpty(nextPageLinkString))
-                    {
-                        response.Value.InitializeNextPageRequest(
-                            this.Client,
-                            nextPageLinkString);
-                    }
-
-                    // Copy the additional data collection to the page itself so that information is not lost
```

Comment:
maybe we should keep that comment for context?

Reply:
Agreed

Reply:
Resolved via https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/512/commits/467f9b74e54be3ca2a8517f7af6cfe1d6039b71d